src/NetMQ/packages.NetMQ.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AsyncIO" version="0.1.18.0" targetFramework="net40" />
-  <package id="JetBrains.Annotations" version="10.0.0" targetFramework="net40" />
+  <package id="JetBrains.Annotations" version="10.0.0" targetFramework="net40" developmentDependency="true"/>
```

Comment:
Interesting and probably worthwhile though has no effect on our current way of packing for NuGet as we list dependencies in the nuspec file.

http://nuget.codeplex.com/workitem/1956

For completeness this might also be included in the .NET 3.5 .config file too.